adoc/chapters/programming_interface.adoc

```diff
@@ -20960,8 +20960,7 @@ a@
 ----
 genfloat logb (genfloat x)
 ----
-   a@ Compute the exponent of x, which is the integral
-      part of logr (latexmath:[|x|]).
+   a@ Compute the integral part of the logarithm of latexmath:[|x|]. The base of the logarithm is the radix (integer base) used by the representation of floating point numbers.
```

Comment:
FWIW,  OpenCL 3.0 has exactly the same wording as SYCL (prior to your PR).  I _think_ this is correct, except that the base `r` isn't defined.  The same function exists in ISO C++, and [cppreference](https://en.cppreference.com/w/cpp/numeric/math/logb) defines it this way:

> Extracts the value of the unbiased radix-independent exponent from the floating-point argument arg, and returns it as a floating-point value
> [...]
> Formally, the unbiased exponent is the signed integral part of `log
 r|arg|` (returned by this function as a floating-point value), for non-zero `arg`, where `r` is `std::numeric_limits<T>::radix` and `T` is the floating-point type of `arg`. If `arg` is subnormal, it is treated as though it was normalized.

For any sane implementation, I think `r` is `2`, but I guess IEEE 754 allows other  radices for floating-point representation?

I think your description is equivalent, except it might be better to replace "the radix (integer base) used by the representation of floating point numbers" with "`std::numeric_limits<T>::radix`, where `T` is the type of `x`".

Reply:
Yes, I think it is OK to explain (in words) what `r` is, but we should also say that it is the value returned by `std::numeric_limits<T>::radix` since that is a more formal definition.

Reply:
> std::numeric_limits<T>::radix

I've just referred to it as std::numeric_limits<T>::radix . I think this is fine. Happy to add a wordy description of you have a suggestion.  Thanks

Reply:
I think this looks good.  I just had a small comment on formatting.